### DISCUSSION TITLE
Unable to keep codespace forever

### DISCUSSION THREAD

**Discussion by mnordstr** [community/community#52504] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hi,

According to the docs it should be possible to keep codespaces forever by clicking on "keep codespace" in the menu next to the codespace in question. I am not seeing this option however, the codespace in question is using my private account but the repository is from an organization. I have added my credit card details under my personal account so billing should be possible. Am I missing something?

Regards,
Mattias

**Comment by joshaber** [comment]
Hi Matthias,

When you go to https://github.com/codespaces is the codespace grouped under your user account or under the repository's organization? That should tell you whether it's billed to you or the organization.

If it's disallowed by the organization's policy you should see "Keep codespace" grayed out, with a message like this:

<img width="385" alt="image" src="https://user-images.githubusercontent.com/13760/231467061-1ab881d3-509a-4f8b-a51f-49b9314030d1.png">

**Reply by mnordstr** [reply]
Hi,

The codespace is grouped under my personal account. No "keep codespace" option there at all.

![image](https://user-images.githubusercontent.com/3437911/231470566-f5cdbbab-2eb7-4cc3-845a-6d2cdcdd3aa8.png)

**Reply by joshaber** [reply]
Aha, there's definitely a bug here. I've opened an issue internally, thanks for letting us know!

**Reply by joshaber** [reply, marked as answer]
@mnordstr could you give it another shot? We believe the bug is fixed.

**Reply by mnordstr** [reply]
Excellent, thanks, now it works! :)
